packages/asset-swapper/src/quote_consumers/exchange_proxy_swap_quote_consumer.ts

```diff
@@ -567,19 +598,30 @@ export class ExchangeProxySwapQuoteConsumer implements SwapQuoteConsumerBase {
         for_loop: for (const [i, order] of quote.orders.entries()) {
             switch_statement: switch (order.source) {
                 case ERC20BridgeSource.Native:
-                    if (order.type !== FillQuoteTransformerOrderType.Rfq) {
+                    if (order.type !== (FillQuoteTransformerOrderType.Rfq || FillQuoteTransformerOrderType.Otc)) {
                         // Should never happen because we check `isMultiplexBatchFillCompatible`
                         // before calling this function.
-                        throw new Error('Multiplex batch fill only supported for RFQ native orders');
+                        throw new Error('Multiplex batch fill only supported for RFQ native orders and OTC Orders');
+                    }
+                    if (order.type !== FillQuoteTransformerOrderType.Rfq) {
+                        subcalls.push({
+                            id: MultiplexSubcall.Rfq,
```

Comment:
Is `id`  backward? 

looks like id === Rfq when `order.type !== FillQuoteTransformerOrderType.Rfq`